src/index.html

```diff
@@ -3,11 +3,296 @@
+            >
+          </a>
+          <a
+            href="tel:+1 234 555-55-55"
+            class="header__call-link"
+          >
+          </a>
+          <a href="#menu" class="header__menu-toggler"></a>
+        </section>
+
+        <h1 class="header__title">Take the Streets</h1>
+      </div>
+    </header>
+    <section class="page__menu menu" id="menu">
+      <div class="menu__content-wrapper">
+        <div class="menu__top-wrapper">
+          <a href="/">
+            <img
+              src="images/logo.png"
+              alt="MyBike Logo"
+              class="menu__logo"
+            >
+          </a>
+          <a href="#" class="menu__toggler"></a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__link--first" >Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#promo" class="nav__link" >About us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#compare" class="nav__link" >Compare bikes</a>
+            </li>
+            <li class="nav__item">
+              <a href="#details" class="nav__link">Details</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contact" class="nav__link" >Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="menu__links-wrapper">
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__link"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__link menu__link--border"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <main class="main">
+      <section class="promo" id="promo">
+        <div class="promo__content-wrapper">
+          <h2>Move Free</h2>
+          <p class="promo__text">
+            Shift your ride, not gears. Find the fastest
+            way to move in the city as the bike adapts
+            intuitively to power the speed you need.
+          </p>
+        </div>
+      </section>
+
+      <section class="main__compare compare" id="compare">
+        <div class="compare__content-wrapper">
+          <h2 class="compare__heading">Compare Bikes</h2>
+
+          <div class="compare__photo-container-wrapper">
+            <div class="compare__photo-container">
+              <img
+                src="images/photos/compare-bike-1.png"
+                alt="Sporty 4 bike"
+                class="compare__photo"
+              >
+              <h3 class="compare__bike-name">Sporty 4</h3>
+              <p class="compare__bike-description">
+                The iconic frame brought to a new performance
+                height as a sporty, active ride.
+              </p>
+              <h3 class="compare__bike-price">$ 2 590</h3>
```

Comment:
@LukaszGryziewicz you can solve the image overflow with position and z-index:
![image overflow text](https://user-images.githubusercontent.com/39308717/163336571-418b1d83-eddb-4c4b-8ffe-8dc7ce5246d7.png)